cryptonight/src/lib.rs

```diff
@@ -178,57 +155,57 @@ mod tests {
         test(
             "5468697320697320612074657374205468697320697320612074657374205468697320697320612074657374",
             "f759588ad57e758467295443a9bd71490abff8e9dad1b95b6bf2f5d0d78387bc",
-            1806260
+            1806260,
```

Comment:
Ah, ok, I see you took the closure tests from here from... @Boog900 https://github.com/Cuprate/cuprate/commit/55b7699a82fb48bfbfd4fabe5e9f3a6b9697bcac.

They can/should still be `fn` though.

Reply:
> They can/should still be fn though.

I'm not sure I understand that last sentence. I left the unit tests in `lib.rs` as they were, as the external API here hasn't changed. These tests are using the expected values from the regression suite in the C++ repo. The extra commas were added by a cargo fmt or clippy command, not because I was trying to introduce extra changes.

The underlying `cn_slow_hash`  that implements all the versions also has a bunch of new tests generated for it which used the C code to get the expected values.

Reply:
I understand the comment now. I converted the preexisting `test` lambdas to nested `test` functions in `lib.rs`. I had modeled the other tests in this crate on these existing tests, that's why they were all using a lambda named `test` instead of a nested function.